dags/app/twitter_post_notification_bot/udf.py

```diff
@@ -0,0 +1,62 @@
+import requests
+from airflow import settings
+from sqlalchemy.orm import sessionmaker
+
+
+def main() -> None:
+    url = "https://twitter135.p.rapidapi.com/v2/UserTweets/"
+    # 499339900 is PyConTW's twitter id
+    querystring = {"id": "499339900", "count": "1"}
+    headers = {
+        "X-RapidAPI-Key": Variable.get("RAPIDAPIAPI_KEY"),
+        "X-RapidAPI-Host": "twitter135.p.rapidapi.com",
+    }
+    webhook_url = Variable.get("DISCORD_POST_NOTIFICATION_WEBHOOK")
+    response = requests.get(url, headers=headers, params=querystring)
+    response_json = response.json()
+    try:
+        Session = sessionmaker(bind=settings.engine)
+        # Update the variable using a context manager
+        variable_key = "TWITTER_LATEST_REST_ID"
+        rest_id = response_json["data"]["user"]["result"]["timeline_v2"]["timeline"][
+            "instructions"
+        ][1]["entries"][0]["content"]["itemContent"]["tweet_results"]["result"][
+            "rest_id"
+        ]
+        full_text = response_json["data"]["user"]["result"]["timeline_v2"]["timeline"][
+            "instructions"
+        ][1]["entries"][0]["content"]["itemContent"]["tweet_results"]["result"][
+            "legacy"
+        ][
+            "full_text"
+        ]
+        rest_id_in_DB = Variable.get(variable_key)
+        if rest_id_in_DB < rest_id:
+            # Create a session
+            session = Session()
+
+            # Query the variable by key
+            variable = session.query(Variable).filter_by(key=variable_key).first()
+
+            # Update the variable value
+            variable.set_val(rest_id)
```

Comment:
update the cursor once we find a bigger `rest_id`. Seems to me that twitter would increment the `rest_id`